langchain/langchain_interceptor.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+import asyncio
+from contextvars import copy_context
+
+from langsmith import get_tracing_context
+from langsmith.run_helpers import _set_tracing_context
+from temporalio import workflow
+
+with workflow.unsafe.imports_passed_through():
+    from contextlib import contextmanager
+    from typing import Any, Mapping, Protocol, Type
+
+    import temporalio.activity
+    import temporalio.api.common.v1
+    import temporalio.client
+    import temporalio.converter
+    import temporalio.worker
+    import temporalio.workflow
```

Comment:
Standard library and `temporalio` modules are automatically passed through, they do not need to be under here